ingest/clusters.py

```diff
@@ -22,13 +22,13 @@ def __init__(self, file_path, *, name: str = None, study_accession: str = "", do
         self.source_file_type = 'cluster'
         self.has_z = 'z' in self.header
         self.top_level_doc = {
```

Comment:
As we discussed, we need to add the `cell_annotations` dictionary to `top_level_doc`.  If it makes more sense to do this in a separate ticket/PR, that's fine so long as it gets added.